admin/tools/docker/shmux/run-multinode-tests.sh

```diff
@@ -7,16 +7,25 @@ set -x
 
 . ./env.sh
 
-shmux -c "docker rm -f qserv; nohup docker run --name qserv --rm --net=host \
+ssh "$MASTER" "docker rm -f qserv; \
+    docker run --detach=true \
+    --name qserv --net=host \
```

Comment:
Since you are using --net=host, I don't think you need the pass-through port mapping below?  I guess it doesn't hurt anything though...